test/client/before-test.js

```diff
@@ -8,6 +8,8 @@
     var hammerhead    = window['%hammerhead%'];
     var nativeMethods = hammerhead.nativeMethods;
 
+    var Promise = hammerhead.Promise;
```

Comment:
You define `Promise` as a global object. It may bring to wrong tests for browsers that doesn't support `Promises` (i.e `IE11`).  Put `Promise` declaration into `window.wait` function.